mail/dovecot/files/dovecot-core-pushnotify.patch

```diff
@@ -876,28 +309,26 @@ diff -Naur ./src/imap/imap-commands.h ./src/imap/imap-commands.h
  bool cmd_x_cancel(struct client_command_context *cmd);
  bool cmd_x_state(struct client_command_context *cmd);
  
-diff -Naur ./src/imap/imap-settings.c ./src/imap/imap-settings.c
---- ./src/imap/imap-settings.c	2020-09-18 20:09:11.000000000 -0400
-+++ ./src/imap/imap-settings.c	2020-09-18 19:57:02.000000000 -0400
-@@ -73,6 +73,7 @@
- 	DEF(SET_STR, imap_logout_format),
- 	DEF(SET_STR, imap_id_send),
- 	DEF(SET_STR, imap_id_log),
-+	DEF(SET_STR, aps_topic),
- 	DEF(SET_ENUM, imap_fetch_failure),
- 	DEF(SET_BOOL, imap_metadata),
- 	DEF(SET_BOOL, imap_literal_minus),
-@@ -101,6 +102,7 @@
+--- ./src/imap/imap-settings.c
++++ ./src/imap/imap-settings.c
+@@ -75,6 +75,7 @@
+ 	DEF(STR, imap_logout_format),
+ 	DEF(STR, imap_id_send),
+ 	DEF(STR, imap_id_log),
++	DEF(STR, aps_topic),
```

Comment:
@essandess: everything else in your own patch is the same as in my file, except here.

Is adding `DEF(STR, aps_topic)` no longer needed then?

Reply:
Oops, thanks for catching that. It's fixed now. Not sure why it builds and works without that definition—I'll go back and sanity check compilation.